tests/integration/conftest.py

```diff
@@ -134,3 +135,12 @@ def __init__(self, sql_backend: SqlBackend, schema: str, tables: list[TableInfo]
 
     def snapshot(self) -> list[Table]:
         return self._tables
+
+
+class StaticTableMapping(TableMapping):
+    def __init__(self, ws: WorkspaceClient, folder: str | None = None, rules: list[Rule] | None = None):
```

Comment:
```suggestion
    def __init__(self, rules: list[Rule]:
```
static mapping doesn't need workspace client.

Reply:
OK